cassandra_repair_scheduler.py

```diff
@@ -262,16 +263,19 @@ def run_repair(self):
                "--dry-run"]     # So we get a list of commands to run.
         if self.option_group.local:
             cmd.append("--local")
+	if self.option_group.parallel:
+	    cmd.append("-p")
         logging.debug("geting repair steps, this may take a while")
-        repair_steps = subprocess.check_output(cmd).split('\n')
+        repair_steps = subprocess.Popen(cmd,stdout=subprocess.PIPE,stderr=subprocess.STDOUT,).communicate()[0].split('\n')
```

Comment:
This seems like a pointless change here, unless you're planning on doing something strange elsewhere.